### DISCUSSION TITLE
v1.0.0.0 Release

### DISCUSSION THREAD

**Discussion by RadzPrower** [ajohns6/Zelda-3-Launcher#1] [Announcements]
### Initial Release
The initial release of the Zelda 3 Launcher tool.

Includes One-Click Build Functionality:
- Download zelda3 repository
- Download dependencies
- Download python
- Download pip
- Install python
- Install pip
- Install dependencies
- Extract ROM resources
- Compile ROM resources
- Compile zelda3.exe

All with a single click.

<hr /><em>This discussion was created from the release <a href='https://github.com/ajohns6/Zelda-3-Launcher/releases/tag/v1.0.0.0'>v1.0.0.0 Release</a>.</em>

**Comment by FreekoTravis** [comment]
I thought the whole point was the rom wasn't included for obvious reasons, won't this just get taken down now?
Or is the rom not actually owned by nin10

**Reply by RadzPrower** [reply]
You are absolutely right and that is why the ROM is not included. The instructions state that you will need to provide your own legitimate copy of the ROM.

The tool is intended to automate the installation of prerequisite software that tends to trip people up. It does not contain the ROM or any of the associated assets.

**Comment by Iceflow420** [comment]
Tried this today, very clean and easy install. My only issue was not being able to get the settings button to activate.

**Reply by RadzPrower** [reply]
It is not functional in v1.0. I wanted to get something out as quick as possible, so I released without it, but I am nearing completion of v1.1 which includes many of the settings normally available via the INI file of zelda3.exe.
![image](https://user-images.githubusercontent.com/37708128/218126426-3c6b80b1-805d-40cb-8cc7-c837f3761572.png)
v1.1 will not include keymapping support, but v1.2 will be along as quick as possible after the release of v1.1. I will have to create a special submenu for it and I'll need to set up keyboard and controller input recognition, (something I've never done) so I am not sure how long it will take to create that menu.

**Comment by FreekoTravis** [comment]
Ok so all I need to do is obtain a normal ROM ( by getting a copy of it I
own and dumping it of course )
